db_management.py

```diff
@@ -1,31 +1,52 @@
-import sqlite3
+from sqlalchemy import create_engine, Column, Integer, String
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
 
 
-def new_user_creation(username_text, pass_text):
-    conn = sqlite3.connect('data_base.db')
-    curs = conn.cursor()
-    params = (username_text, pass_text)
-    curs.execute("INSERT INTO users VALUES (NULL, ?, ?)", params)
-    conn.commit()
-    conn.close()
+engine = create_engine('sqlite:///data_base.db')
```

Comment:
Let's create a separate configuration file and put stuff like connection strings inside this file. Then you can use this value from that file.

The rational behind this is configuration changes often, and we don't want to change the code for configuration changes.
Also, you never know if you'll need this value again someplace else. It's better to use it in one central place rather than scattered across your project.